roles/rook/defaults/main.yml

```diff
@@ -0,0 +1,153 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+configuration_directory: /opt/configuration
+
+##########################
+# k3s/metallb
+
+metallb_rook_external_IP: 192.168.16.101
+
+##########################
+# rook
+
+container_registry_rook: quay.io
+rook_ceph_image: "{{ container_registry_rook }}/ceph/ceph"
+# renovate: datasource=docker depName=quay.io/rook/ceph
+rook_ceph_image_tag: "v17.2"
+
+## template directory containing rook CRD `*.yml.j2` files, e.g. OSISM configuration directory
+rook_template_directory: "{{ role_path }}/templates"
+rook_configuration_directory: "{{ configuration_directory }}/environments/rook"
+## kubernetes namespace in which the rook cluster should be installed
+## also see https://rook.io/docs/rook/v1.13/Storage-Configuration/Advanced/ceph-configuration/?h=#using-alternate-namespaces
+rook_namespace: rook-ceph
+
+## work directory inside the osism-ansible container
+rook_work_directory: /tmp/rook/configuration
+
+## run cleanup tasks and delete cluster
+rook_cleanup: false
+
+##########################
+# rook CRD
+# have a look at `templates/*.yml.j2` to see where exaclty the variables are applied
+# for ultimate customization, change `{{ rook_template_directory }}` to point to your configuration repository
+
+## rook cluster name
+rook_cluster_name: rook-ceph
+rook_mon_count: 3
+rook_mds_count: 3
+rook_mgr_count: 3
+rook_mgr_modules:
+  - name: balancer
+    enabled: true
+  - name: status
+    enabled: true
+  - name: prometheus
+    enabled: true
+  # obtain the information about physical disks in dashboard
+  - name: rook
+    enabled: true
+rook_dashboard_enabled: true
+rook_dashboard_ssl: true
+rook_dashboard_port: 7000
+rook_dashboard_port_external: 443
+# https://rook.io/docs/rook/latest-release/Storage-Configuration/Monitoring/ceph-monitoring/
+rook_monitoring_enabled: false # TODO: enable 
```

Comment:
@matofeder It would be great if you helped out with monitoring parts! Rook has quite some documentation on this.
But I guess this can be done after merging this initial MR.

Reply:
I'm in the process of adding CEPH monitoring support to the SCS monitoring (kubernetes-monitoring play). The next stage will be to port the support to the OSISM, where we can cooperate. 

 > But I guess this can be done after merging this initial MR.

Sure this is not a blocker for this PR. I just wanted to let you know that the Prometheus-based monitoring is already there in the OSISM plays, and the CEPH-related part of monitoring is ongoing.